content/en/install.md

```diff
@@ -3,101 +3,222 @@ title: Installation
+
+        pixi add scipy
+
+4.  Try out SciPy!
+
+        pixi run python
+
+See next steps in [the SciPy user guide][scipy-user-guide].
+
+<a name="environment-based"></a>
+
+## Environment-based workflows
+
+In project-based workflows, a project is a directory containing a manifest
+file describing the project, a lock-file describing the exact dependencies
+of the project, and the project's (potentially multiple) environments.
+
+In contrast,
+in environment-based workflows you install packages into an environment,
+which you can activate and deactivate from any directory.
+These workflows are well-established,
+but lack some reproducibility benefits of project-based workflows.
+
+### Installing with `pip`
+
+1.  [Install Python](https://www.python.org/downloads/).
+
+2.  Create and activate a virtual environment with `venv`.
+
+    {{< admonition hint >}}
+See [the tutorial in the Python Packaging User Guide](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments).
+    {{< /admonition >}}
+
+3.  Install SciPy, using [`pip`]:
+
+        python -m pip install scipy
+
+[`pip`]: https://pip.pypa.io/en/stable/getting-started/
+
+### Installing with `conda`
+
+[Miniforge] is the recommended way to install `conda` and [`mamba`],
+two Conda-based environment managers.
+After creating an environment, you can install SciPy from conda-forge as follows:
+
+    conda install scipy # or
+    mamba install scipy
+
+[Miniforge]: https://conda-forge.org/download/
+[`mamba`]: https://mamba.readthedocs.io/en/latest/
+
+<a name="system-package-managers"></a>
+
+## Installing system-wide via a system package manager
 
 System package managers can install the most common Python packages.
 They install packages for the entire computer, often use older versions,
-and don't have as many available versions.
+and don't have as many available versions. They are not the recommended
+installation method.
 
-## Ubuntu and Debian
+### Ubuntu and Debian
 
 Using `apt-get`:
 
     sudo apt-get install python3-scipy
 
-## Fedora
+### Fedora
 
 Using `dnf`:
 
     sudo dnf install python3-scipy
 
-## macOS
+### macOS
 
 macOS doesn't have a preinstalled package manager, but you can install
 [Homebrew](https://brew.sh/) and use it to install SciPy (and Python itself):
 
     brew install scipy
 
-<a name="source"></a>
+<a name="building-from-source"></a>
 
-# Source packages
+## Building from source
 
 A word of warning: building SciPy from source can be a nontrivial exercise. We
-recommend using binaries instead if those are available for your platform.
+recommend using binaries instead if those are available for your platform
+via one of the above methods.
 For details on how to build from source, see
-[this guide in the SciPy docs](https://scipy.github.io/devdocs/building/index.html).
+[the building from source guide in the SciPy docs][building-docs].
+
+[building-docs]: https://scipy.github.io/devdocs/building/index.html
```

Comment:
Given the default recommendation to use `uv` for projects, I was wondering whether, in the future, we should adjust the above page accordingly?

I presume it can be used the same way pip is for SciPy development, adding a third code tab with `uv run python dev.py` and `uv pip` for installing dependencies?

Reply:
> Given the default recommendation to use uv for projects, I was wondering whether, in the future, we should adjust the above page accordingly?

Yep! Although for SciPy, given the compiler dependencies, Pixi will probably be our best bet. I already use https://github.com/rgommers/pixi-dev-scipystack for all of my SciPy development. Windows support for PyTorch and JAX on conda-forge is still maturing (though actively - https://github.com/conda-forge/pytorch-cpu-feedstock/pull/231), but we may be able to use PyPI on Windows to fill some gaps if anyone needs it.

A third tab for `uv` is probably not so useful, since it's relatively obvious what to do by following along with the `pip` instructions, and you have to do the same setup of system deps.

> I presume it can be used the same way pip is for SciPy development, adding a third code tab with uv run python dev.py and uv pip for installing dependencies?

All that to say that we could do something like this. Or we could make a new page instead of adding a new tab, since there is much less to explain when most of the work is already done in writing the Pixi manifest. We may want to wait until somebody wants to try to merge Pixi support into the main repo, though.

Reply:
Thanks, Lucas!